src/content/documentation/docs/rqb.mdx

```diff
@@ -403,6 +403,31 @@ export const posts = pgTable('posts', {
 });
 ```
 
+For constraints specified with the `foreignKey` operator, foreign key actions are defined with the syntax:
+
+```typescript {18-19}
+import { foreignKey, pgTable, serial, text, integer } from 'drizzle-orm/pg-core';
+
+export const users = pgTable('users', {
+	id: serial('id').primaryKey(),
+	name: text('name'),
+});
+
+export const posts = pgTable('posts', {
+	id: serial('id').primaryKey(),
+	name: text('name'),
+	author: integer('author').notNull(),
+}, (table) => {
+	fk: foreignKey({
```

Comment:
It's missing a return statement in the callback function.

```ts
// ...
 (table) => ({
    fk: foreignKey({
      name: 'author_fk',
      columns: [table.author],
      foreignColumns: [users.id],
    })
      .onDelete('cascade')
      .onUpdate('cascade'),
  }),
```

Reply:
done, I've fixed the indentation and added a trailing comma to match the other examples as well.